tox.ini

```diff
@@ -19,7 +19,7 @@ commands_pre =
     pip install -e plugins/dlab_aws
```

Comment:
I do not think you needs to use plugins here at all

Reply:
removed